exo-go/src/app_dependency_helpers/index.go

```diff
@@ -0,0 +1,18 @@
+package appDependencyHelper
+
+import (
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+type dependency interface {
+	GetDeploymentConfig() map[string]string
+}
+
+func Build(dependencyConfig types.Dependency, appConfig types.AppConfig) dependency {
+	switch dependencyConfig.Name {
+	case "exocom":
+		return ExocomDependency{dependencyConfig, appConfig}
+	default:
+		return GenericDependency{dependencyConfig, appConfig}
+	}
+}
```

Comment:
@cheahuychou I assume you will be implementing something like this in your `exo setup` work. Let's make sure they line up.